### DISCUSSION TITLE
Where is the training log?

### DISCUSSION THREAD

**Discussion by myrainbowandsky** [voicepaw/so-vits-svc-fork#627] [Q&A]
**Describe the bug**
System: Ubuntu 22.04
I used remote ssh command line to train the model. It was interrupted and when I log again, the processing is still  running, but I cannot check the training log. How to find it ?

**Comment by 34j** [comment]
There is no way to see the logs, as we have stopped logging to files because it is dangerous to do so (because of multiprocessing).
Use of tmux or similar is recommended.
